### DISCUSSION TITLE
Kudos to being featured on Linus Tech Tips!

### DISCUSSION THREAD

**Discussion by kylegordy** [blakeblackshear/frigate#1912] [General]
I spotted this while eating lunch today:

[![Linus Tech Tips Video Featuring Frigate](https://img.youtube.com/vi/B635wcdr6-w/0.jpg)](https://www.youtube.com/watch?v=B635wcdr6-w)

Keep up the awesome work y'all 🤘

**Comment by blackgold9** [comment]
Just saw that myself. They loved it!

**Comment by kitchung** [comment]
I came here to mention this video too! Man, that card is overkill though.
